### ISSUE TITLE
ERROR:  .[cuda-12] is not a valid editable requirement.

### ISSUE THREAD

**Issue by M-Mahou** [LykosAI/ComfyUI-Inference-Core-Nodes#1]
When trying to run `ComfyUI-Inference-Core-Nodes/install.py`

I get this error 

```
ERROR:  .[cuda-12] is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with bzr+http, bzr+https, bzr+ssh, bzr+sftp, bzr+ftp, bzr+lp, bzr+file, git+http, git+https, git+ssh, git+git, git+file, hg+file, hg+http, hg+https, hg+ssh, hg+static-http, svn+ssh, svn+http, svn+https, svn+svn, svn+file).
Traceback (most recent call last):
  File "/notebooks/ComfyUI/custom_nodes/ComfyUI-Inference-Core-Nodes/install.py", line 37, in <module>
    main()
  File "/notebooks/ComfyUI/custom_nodes/ComfyUI-Inference-Core-Nodes/install.py", line 28, in main
    pip_install("-e .[cuda-12]")
  File "/notebooks/ComfyUI/custom_nodes/ComfyUI-Inference-Core-Nodes/install.py", line 10, in pip_install
    subprocess.check_call(
  File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/usr/bin/python3', '-m', 'pip', 'install', '-e .[cuda-12]']' returned non-zero exit status 1.
```

Note that im using linux and not windows, as 

cuda-12 = [
    "onnxruntime-gpu @ https://aiinfra.pkgs.visualstudio.com/2692857e-05ef-43b4-ba9c-ccf1c22c437c/_packaging/9387c3aa-d9ad-4513-968c-383f6f7f53b8/pypi/download/onnxruntime-gpu/1.17.1/onnxruntime_gpu-1.17.1-cp310-cp310-win_amd64.whl ; platform_system == 'Windows'"
]

replacing the link with 
onnxruntime-gpu @ https://aiinfra.pkgs.visualstudio.com/PublicPackages/_apis/packaging/feeds/9387c3aa-d9ad-4513-968c-383f6f7f53b8/pypi/packages/onnxruntime-gpu/versions/1.17.1/onnxruntime_gpu-1.17.1-cp310-cp310-manylinux_2_28_x86_64.whl/content

Gave the same error, so i just removed the -e

**Comment by harbinman** [comment]
i have

> When trying to run `ComfyUI-Inference-Core-Nodes/install.py`
> 
> I get this error
> 
> ```
> ERROR:  .[cuda-12] is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with bzr+http, bzr+https, bzr+ssh, bzr+sftp, bzr+ftp, bzr+lp, bzr+file, git+http, git+https, git+ssh, git+git, git+file, hg+file, hg+http, hg+https, hg+ssh, hg+static-http, svn+ssh, svn+http, svn+https, svn+svn, svn+file).
> Traceback (most recent call last):
>   File "/notebooks/ComfyUI/custom_nodes/ComfyUI-Inference-Core-Nodes/install.py", line 37, in <module>
>     main()
>   File "/notebooks/ComfyUI/custom_nodes/ComfyUI-Inference-Core-Nodes/install.py", line 28, in main
>     pip_install("-e .[cuda-12]")
>   File "/notebooks/ComfyUI/custom_nodes/ComfyUI-Inference-Core-Nodes/install.py", line 10, in pip_install
>     subprocess.check_call(
>   File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
>     raise CalledProcessError(retcode, cmd)
> subprocess.CalledProcessError: Command '['/usr/bin/python3', '-m', 'pip', 'install', '-e .[cuda-12]']' returned non-zero exit status 1.
> ```
> 
> Note that im using linux and not windows, as
> 
> cuda-12 = [ "onnxruntime-gpu @ https://aiinfra.pkgs.visualstudio.com/2692857e-05ef-43b4-ba9c-ccf1c22c437c/_packaging/9387c3aa-d9ad-4513-968c-383f6f7f53b8/pypi/download/onnxruntime-gpu/1.17.1/onnxruntime_gpu-1.17.1-cp310-cp310-win_amd64.whl ; platform_system == 'Windows'" ]
> 
> replacing the link with onnxruntime-gpu @ https://aiinfra.pkgs.visualstudio.com/PublicPackages/_apis/packaging/feeds/9387c3aa-d9ad-4513-968c-383f6f7f53b8/pypi/packages/onnxruntime-gpu/versions/1.17.1/onnxruntime_gpu-1.17.1-cp310-cp310-manylinux_2_28_x86_64.whl/content
> 
> Gave the same error, so i just removed the -e

i have same error did you fix that?

**Comment by harbinman** [comment]
> When trying to run `ComfyUI-Inference-Core-Nodes/install.py`
> 
> I get this error
> 
> ```
> ERROR:  .[cuda-12] is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with bzr+http, bzr+https, bzr+ssh, bzr+sftp, bzr+ftp, bzr+lp, bzr+file, git+http, git+https, git+ssh, git+git, git+file, hg+file, hg+http, hg+https, hg+ssh, hg+static-http, svn+ssh, svn+http, svn+https, svn+svn, svn+file).
> Traceback (most recent call last):
>   File "/notebooks/ComfyUI/custom_nodes/ComfyUI-Inference-Core-Nodes/install.py", line 37, in <module>
>     main()
>   File "/notebooks/ComfyUI/custom_nodes/ComfyUI-Inference-Core-Nodes/install.py", line 28, in main
>     pip_install("-e .[cuda-12]")
>   File "/notebooks/ComfyUI/custom_nodes/ComfyUI-Inference-Core-Nodes/install.py", line 10, in pip_install
>     subprocess.check_call(
>   File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
>     raise CalledProcessError(retcode, cmd)
> subprocess.CalledProcessError: Command '['/usr/bin/python3', '-m', 'pip', 'install', '-e .[cuda-12]']' returned non-zero exit status 1.
> ```
> 
> Note that im using linux and not windows, as
> 
> cuda-12 = [ "onnxruntime-gpu @ https://aiinfra.pkgs.visualstudio.com/2692857e-05ef-43b4-ba9c-ccf1c22c437c/_packaging/9387c3aa-d9ad-4513-968c-383f6f7f53b8/pypi/download/onnxruntime-gpu/1.17.1/onnxruntime_gpu-1.17.1-cp310-cp310-win_amd64.whl ; platform_system == 'Windows'" ]
> 
> replacing the link with onnxruntime-gpu @ https://aiinfra.pkgs.visualstudio.com/PublicPackages/_apis/packaging/feeds/9387c3aa-d9ad-4513-968c-383f6f7f53b8/pypi/packages/onnxruntime-gpu/versions/1.17.1/onnxruntime_gpu-1.17.1-cp310-cp310-manylinux_2_28_x86_64.whl/content
> 
> Gave the same error, so i just removed the -e

all fixed?

**Comment by thecooltechguy** [comment]
Fixed this bug in this pr: #2 , hopefully it will get merged!
